vochain/indexer/migrations/0005_create_table_token_transfers.sql

```diff
@@ -0,0 +1,17 @@
+-- +goose Up
+CREATE TABLE token_transfers (
+  tx_hash BLOB NOT NULL PRIMARY KEY,
+  height INTEGER NOT NULL,
+  from_account BLOB NOT NULL,
+  to_account BLOB NOT NULL,
+  amount INTEGER NOT NULL,
+  transfer_time DATETIME NOT NULL
+);
+
+CREATE INDEX index_from_account_token_transfers
+ON token_transfers(from_account);
+
+-- +goose Down
+DROP TABLE token_transfers
+
+DROP INDEX index_from_account_token_transfers
```

Comment:
nit: include trailing newlines. perhaps configure your editor for that?

Reply:
I've just missed it :S